aQute.libg/src/aQute/lib/collections/MultiMap.java

```diff
@@ -73,6 +73,10 @@ public boolean add(K key, V value) {
 
 	@SuppressWarnings("unchecked")
 	public boolean addAll(K key, Collection< ? extends V> value) {
+
+		if (value == null)
+			return false;
+
```

Comment:
This seems a bit odd. Do you not allow null values? It seems you either allow null values or you don't. If you don't then throw an exception rather then return false.